.golangci.yml

```diff
@@ -49,7 +49,7 @@ linters-settings:
 
       # Some configured revive rules
       - name: imports-blacklist
-        arguments: ["errors", "github.com/pkg/errors"] # Prefer ./app/errors
+        arguments: ["errors", "github.com/pkg/errors","github.com/golang/protobuf"] # Prefer ./app/errors
```

Comment:
got some warnings about "github.com/golang/protobuf" being deprecated, so fixed that as well.